mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergFilterFactory.java

```diff
@@ -82,10 +80,12 @@ public void testNotEqualsOperand() {
     UnboundPredicate childExpressionActual = (UnboundPredicate) actual.child();
     UnboundPredicate childExpressionExpected = Expressions.equal("salary", 3000L);
 
-    assertEquals(actual.op(), expected.op());
-    assertEquals(actual.child().op(), expected.child().op());
-    assertEquals(childExpressionActual.ref().name(), childExpressionExpected.ref().name());
-    assertEquals(childExpressionActual.literal(), childExpressionExpected.literal());
+    Assertions.assertThat(expected.op()).isEqualTo(actual.op());
```

Comment:
it's ok to statically import `assertThat()` here and in the other files

Reply:
Yes Got it.